prov/src/main/java/org/bouncycastle/jcajce/provider/digest/SM3.java

```diff
@@ -29,6 +33,27 @@ public Object clone()
         }
     }
 
+    /**
+     * SHA1 HMac
```

Comment:
@dghgit sorry.. I was forget change javadoc, plz help me modify this, tks! 😃

Reply:
I guess I forgot to read it! Should be updated soon.